.ci/workspace-startup/che7_workspace.json

```diff
@@ -0,0 +1,69 @@
+{
```

Comment:
@ScrewTSW why Che 7 devfile is not used ?

Reply:
same as above, but I would do that in a separate issue, like the documentation.